### PR TITLE
docs: Remove references to sources & clarify update frequency

### PR DESCRIPTION
**Pull request by emmambd**
In preparation for #291, this PR replaces the term "source" with "feed" where "source" in schema descriptions and throughout the README. Also includes information about update frequency as requested originally in https://github.com/MobilityData/mobility-database-catalogs/issues/280.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,92 +2,97 @@
...
 
-The Mobility Database Catalogs is a list of open mobility data sources from across the world. [You can learn more about the Mobility Database here](https://database.mobilitydata.org/).
+The Mobility Database Catalogs is a list of open mobility data feeds from across the world. [You can learn more about the Mobility Database here](https://database.mobilitydata.org/).
 
-To search sources easily, you can download [the CSV spreadsheet](#browsing-and-consuming-the-spreadsheet). If you want to filter for specific types of sources, [you can learn how to here](#get-and-filter-sources).
+To search feeds easily, you can download [the CSV spreadsheet](https://bit.ly/catalogs-csv). If you want to filter for specific types of feeds, [you can learn how to here](#get-and-filter-feeds).
```
Comment (fredericsimard) [inline README.md:7]: I would suggest using a new short link that begins with share.mobilitydata.org.

File: README.md [hunk 2]
```diff
@@ -2,92 +2,97 @@
...
-* [Get and Filter Sources](#get-and-filter-sources)
+* [Get and Filter Feeds](#get-and-filter-feeds)
 * [Integration Tests](#integration-tests)
 * [License](#license)
 * [Contributing](#contributing)
 
 ## Browsing and Consuming The Spreadsheet
 
-If you're only interested in browsing the sources or pulling all the latest URLs, [download the CSV](https://bit.ly/catalogs-csv). You can cross reference IDs from the Mobility Database, TransitFeeds and Transitland with [this ID map spreadsheet](https://docs.google.com/spreadsheets/d/1Q96KDppKsn2khdrkraZCQ7T_qRSfwj7WsvqXvuMt4Bc/edit?resourcekey#gid=1787149399).
+If you're only interested in browsing the feeds or pulling all the latest URLs, [download the CSV](https://bit.ly/catalogs-csv). You can cross reference IDs from the Mobility Database, TransitFeeds and Transitland with [this ID map spreadsheet](https://docs.google.com/spreadsheets/d/1Q96KDppKsn2khdrkraZCQ7T_qRSfwj7WsvqXvuMt4Bc/edit?resourcekey#gid=1787149399).
```
Comment (fredericsimard) [inline README.md:25]: @emmambd : same here

File: README.md [hunk 3]
```diff
@@ -2,92 +2,97 @@
...
 |- authentication_info_url | URL| Conditionally required | If authentication is required, the **authentication_info_url** field contains a URL to a human-readable page describing how the authentication should be performed and how credentials can be created. This field is required for `authentication_type=1` or greater. |
 |- api_key_parameter_name |Text|Conditionally required | The **api_key_parameter_name** field defines the name of the parameter to pass in the URL to provide the API key. This field is required for `authentication_type=1` and `authentication_type=2`.   |  
 |- license_url  |URL| Optional     | The license information for  `direct_download_url`.
 
-In [the CSV](https://bit.ly/catalogs-csv), realtime sources include the location metadata of their static reference when provided.
+In [the CSV](https://bit.ly/catalogs-csv), realtime feeds include the location metadata of their static reference when provided.
```
Comment (fredericsimard) [inline README.md:95]: Same. Replace with : https://share.mobilitydata.org/catalogs-csv

File: README.md [hunk 4]
```diff
@@ -2,92 +2,97 @@
 
 [![Integration tests](https://github.com/MobilityData/mobility-catalogs/actions/workflows/integration_tests.yml/badge.svg?branch=issue%2F343%2Fcatalogs-prototype)](https://github.com/MobilityData/mobility-catalogs/actions/workflows/integration_tests.yml) [![Unit tests](https://github.com/MobilityData/mobility-catalogs/actions/workflows/unit_tests.yml/badge.svg?branch=issue%2F343%2Fcatalogs-prototype)](https://github.com/MobilityData/mobility-catalogs/actions/workflows/unit_tests.yml) [![Export catalogs to CSV](https://github.com/MobilityData/mobility-catalogs/actions/workflows/export_to_csv.yml/badge.svg?branch=issue%2F343%2Fcatalogs-prototype)](https://github.com/MobilityData/mobility-catalogs/actions/workflows/export_to_csv.yml) [![Join the MobilityData chat](https://badgen.net/badge/slack/%20/green?icon=slack)](https://bit.ly/mobilitydata-slack)
```
Comment (fredericsimard) [inline README.md:3]: Replace https://bit.ly/mobilitydata-slack with https://share.mobilitydata.org/slack

**Review by fredericsimard — COMMENTED** [review]
I've left a few suggestions...

File: README.md (hunk 3 above)
Comment (emmambd) [reply README.md:95]: It's important that this URL stays consistent, since this CSV is ingested.

Comment (fredericsimard) [reply README.md:95]: Ok, just a suggestion, we're trying to move away from bit.ly, some we can't even change.

**Review by fredericsimard — APPROVED** [review]

**Review by fredericsimard — APPROVED** [review]

**Pull request merged**